Resources/views/MediaAdmin/edit.html.twig

```diff
@@ -50,8 +67,7 @@ file that was distributed with this source code.
                             <table class="table">
                                 <tr>
                                     <th>{{ 'label.size'|trans({}, 'SonataMediaBundle') }}</th>
-                                    <td>{{ object.width|number_format_decimal }}px x {{ object.height|number_format_decimal }}px
-                                        {% if object.size > 0 %}({{ object.size|number_format_decimal }}o){% endif %}</td>
+                                    <td>{{ object.width }} x {{ object.height }} {% if object.size > 0 %}({{ macros.bytesToSize(object.size) }}){% endif %}</td>
```

Comment:
As stated in the docs, `number_formats` default can be changed globally, which means you should have the right output for each country if the user configured it properly (maybe there is even automatic configuration based on the locale?).

Ok for dropping `px`, it's indeed a bit pedantic.